### ISSUE TITLE
Determine whether automerging can be made to work when fork branches are behind as well as ahead of main

### ISSUE THREAD

**Issue by dylanhmorris** [CDCgov/covid19-forecast-hub#77]
If not, add guidance to make sure branch is strictly ahead of main for PR automerge

Compare: 
https://github.com/CDCgov/covid19-forecast-hub/actions/runs/11945156368 (failed)
https://github.com/CDCgov/covid19-forecast-hub/actions/runs/11945068385 (succeeded)

**Comment by sbidari** [comment]
I have not found a way to make automerge work when fork branches are behind main, I can look into it more but I think the easiest thing to do at least for now is enable **Always suggest updating pull request branches** in the repo settings which will suggest teams to sync their branch with main if it's behind.

**Comment by dylanhmorris** [comment]
Makes sense. I would also add a note to the documentation in the same area as the bit on automerging

**Comment by dylanhmorris** [comment]
@sbidari added notes in #87. I think we should keep this open for a little while to explore whether things can work more automatically with reconcilable (but divergent) branches.

**Comment by sbidari** [comment]
I agree!
